modules/ROOT/pages/quotas-and-limits.adoc

```diff
@@ -35,9 +35,8 @@ Duplicate prompt names or instructions are not allowed.
 | Prompt Response Limits 
 a| 
 * 128 characters for IDP NLP prompts.
-* 4096 output tokens for Einstein prompts. 
-+
-An output token is a text fragment, such as a word or part of a word.
+* 4096 output words for Einstein prompts using `GPT-4o (gpt-4o-2024-05-13)`.
+* 16382 output words for Einstein prompts using `GPT-4o	(gpt-4o-2024-08-06)` or `GPT-4o Mini (gpt-4o-mini-2024-07-18)`.
```

Comment:
Given 1 token ~= 0.75 words, I think it's more accurate to say:
Approximately **3,000** words using GPT-4o (gpt-4o-2024-05-13).
Approximately **12,000** words using GPT-4o (gpt-4o-2024-08-06) or GPT-4o Mini (gpt-4o-mini-2024-07-18).